pkg/util/logger/logger.go

```diff
@@ -57,6 +57,11 @@ func (p Prm) Reload() error {
 	return p._log.reload(p)
 }
 
+// Name returns component name.
+func (p Prm) Name() string {
```

Comment:
This is an applicaiton logic, lets implement this on some adapter in `neofs-node`? Current package is for general purpose.

Reply:
Removed.